wrap_js/cfd_utils.ts

```diff
@@ -1,50 +1,75 @@
-import * as cfdjs from "cfd-js";
+import * as cfdjs from "cfd-js-wasm";
 
-export function GetPubkeyFromPrivkey(privkey: string) {
+let cfd: cfdjs.Cfdjs;
+let initFlag = false;
+
+cfdjs.addInitializedListener(async () => {
+  initFlag = true;
+  cfd = cfdjs.getCfd();
+});
+
+function Sleep(timeout: number) {
+  return new Promise((resolve, reject) => setTimeout(resolve, timeout));
+}
+
+export async function WaitInitialized() {
```

Comment:
I feel a nicer way instead of calling that everywhere would be to transform this util module into a class and check in the constructor whether it is initialized, and throw if it is not. (I don't know why `hasLoadedWasm` is not exposed in the ts definition file of `cfd-js-wasm`?

Reply:
>I don't know why hasLoadedWasm is not exposed in the ts definition file of cfd-js-wasm?

It seems that the addition to d.ts was omitted because it was an API added later. Add it to cfd-js-wasm and then modify it. Thanks!